src/main/java/io/codiga/plugins/jetbrains/Constants.java

```diff
@@ -40,4 +40,7 @@
     public static final long REAL_TIME_FEEDBACK_TIMEOUT_MILLIS = 15 * 1000 ; // 15 seconds
     public static final long SLEEP_BETWEEN_FILE_ANALYSIS_MILLIS = 200;
     public static final long FILE_ANALYSIS_INITIAL_SLEEP_MILLIS = 1200;
+
+    public static final String GET_FILENAME = "&[GET_FILENAME]";
```

Comment:
Let's move these specific constants to a separate file such as `CodingAssistantVariables.java`